m/main_glider_data_processing_dt.m

```diff
@@ -506,42 +557,56 @@
       disp(getReport(exception, 'extended'));
     end
   end
+  
 
+% $$$   %% Generate L1 NetCDF file (processed data), if needed and possible.
+% $$$   if ~isempty(fieldnames(data_processed)) && ~isempty(netcdf_l1_file)
+% $$$     disp('Generating NetCDF L1 output...');
+% $$$     try
+% $$$       outputs.netcdf_l1 = generateOutputNetCDF( ...
+% $$$         netcdf_l1_file, data_processed, meta_processed, deployment, ...
+% $$$         netcdf_l1_options.variables, ...
+% $$$         netcdf_l1_options.dimensions, ...
+% $$$         netcdf_l1_options.attributes);
+% $$$       disp(['Output NetCDF L1 (processed data) generated: ' ...
+% $$$             outputs.netcdf_l1 '.']);
+% $$$     catch exception
+% $$$       disp(['Error generating NetCDF L1 (processed data) output ' ...
+% $$$             netcdf_l1_file ':']);
+% $$$       disp(getReport(exception, 'extended'));
+% $$$     end
+% $$$   end
```

Comment:
Again, just remember that you can disable the netcdf l1 generation in configDataDTPathsLocal.